splice_spark2/src/test/scala/com/splicemachine/spark2/splicemachine/transaction/TransactionIT.scala

```diff
@@ -0,0 +1,367 @@
+/*
+ * Copyright (c) 2012 - 2021 Splice Machine, Inc.
+ *
+ * This file is part of Splice Machine.
+ * Splice Machine is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3, or (at your option) any later version.
+ * Splice Machine is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License along with Splice Machine.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.splicemachine.spark2.splicemachine.transaction
+
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.{FunSuite, Matchers}
+
+@RunWith(classOf[JUnitRunner])
+class TransactionIT extends FunSuite with TestContext with Matchers {
```

Comment:
Is it possible to add tests for savepoint functionality?

Reply:
Added savepoint tests in commit https://github.com/splicemachine/spliceengine/pull/5322/commits/d544e51630aaca9c652ae870f1a22b61b7e0db97